### ISSUE TITLE
KSPWheelDustCamera: getCameraColor() uses only the first pixel of the image

### ISSUE THREAD

**Issue by GER-Space** [shadowmage45/KSPWheel#53]
for (int i = 0; i < len; i++)
            {
                r += cols[0].r;
                g += cols[0].g;
                b += cols[0].b;
            }

Shouldn't this be 
```
            for (int i = 0; i < len; i++)
            {
                r += cols[i].r;
                g += cols[i].g;
                b += cols[i].b;
            }
```
Because now you just could return the first pixel without all the math.

**Comment by shadowmage45** [comment]
Good find.  I had mostly ported that code over from the origin KF-DustFX implementation, and apparently didn't look too close at some of it.

Should be quite simple to fix this up for the next release.  Not sure if I want to simplify it to a single pixel lookup (as I think that was the original intent of the code), or do the actual color averaging.  Since GetPixels() has to be called either way, color averaging would probably be perfectly reasonble for performance (the GetPixels() call will be far far more expensive than summing some floats in a loop).

**Comment by shadowmage45** [comment]
Should be fixed up for the KSP 1.4 / 1.4.1 releases.  Have updated it to use the actual average color from the entire texture rather than just a single pixel.

Thanks for noticing and reporting the problem.
